### PR TITLE
Site updates

### LINKED ISSUE #146: Site updates
NOTE: ASK AROUND + FIGURE OUT WHERE TO GET THE LINKS FROM AS SOON AS YOU GET THIS TICKET!!!! B/c it takes people time to get back to you and what not but TRACK THEM DOWN!!!!

- [x] Change this section to just have Project Guide, Beginner Resource Guide, and Hacker Guide (based on what we discussed in TL) 
Screenshot 2023-01-27 at 11 54 58 PM

- [x] Move meet the team to be below mentors 
- [x] Get the links for: the "Join our Slack" button, "Add to Calendar" button, all of the resources (project guide, beginner resource guide, and hacker guide)
- [x] Fix the nav bar at this screensize (maybe make the text smaller)
Screenshot 2023-01-27 at 11 58 03 PM

### PR DESCRIPTION
**Pull request by Dhivas-Sugumar**
Completed a few site updates and added links.

Fixes #146 .

Changelist:

-Added changes in the ticket. 

Tested:

- on localhost on mobile and desktop.

Screenshots & Screencasts:

image

image

### CODE REVIEW

File: live-site/components/resources-section/ResourcesSection.tsx [hunk 1]
```diff
@@ -7,19 +8,27 @@ import {
 } from './ResourcesSection.styles';
 
 const ResourcesSection: React.FC = () => {
-  const labels: string[] = [
-    'Project Guide',
-    'Judging Process Guide',
-    'Project Demo Guide',
-    'Beginner Resource Guide'
+  const resources: ResourceBoxProps[] = [
```
Comment (jzhzhang66) [inline live-site/components/resources-section/ResourcesSection.tsx]: Could probably move this to the data.ts file

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Lookin good dhivas!!!

The nav bar is still having issues tho. Honestly, I say just remove the "Challenges" link from the nav bar because it's literally at the top of the page lol. And it's not a super important section - its just for fun
Screenshot 2023-02-02 at 11 41 28 PM

**Review by mike10911 — CHANGES_REQUESTED** [review]
Beautiful!!!! Just a couple of changes!

At 1031-1025 the nav links are still shifted down
Screen Shot 2023-02-02 at 11 48 26 PM

Again at 803-768
Screen Shot 2023-02-02 at 11 48 26 PM

Around 1034 the is some container overflow. Maybe add some more padding or smaller text
Screen Shot 2023-02-02 at 11 54 40 PM

Maybe some more padding between sections
Screen Shot 2023-02-02 at 11 56 48 PM

**Review by jzhzhang66 — APPROVED** [review]
LGTM

**Review by mike10911 — APPROVED** [review]
APPROVING with 1 small change

I am still seeing this overflow
Screen Shot 2023-02-03 at 4 26 58 PM

**Pull request merged**